### ISSUE TITLE
[Other] No VA.gov Experience Standard for the issue found. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#50099]
#### VFS team name
Sitewide Content

#### VFS product name
Education rates pages

#### VFS feature name
Education rates pages

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Several links on both pages have `title` text that doesn't seem to add any value. The `title` attribute should be used on links only when the extra text in the `title` attribute clarifies the purpose of the link for screen reader users -- and usually, the better option is just writing unambiguous visible link text and not worry about the `title` attribute at all. On these pages, I'm mainly seeing redundant `title` text, which is a bit of a headache for screen reader users.

### Link, screenshot or steps to recreate
Two examples:

On the Post-9/11 GI Bill Rates page, under "Full rates for school and training programs" > "Added payments," we have:
```
<a ... href="/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program" title="Yellow Ribbon Program">Learn about the Yellow Ribbon Program</a>
``` 
Many screen readers will announce both the visible text and the title text. For example, NVDA announces this as `Learn about the Yellow Ribbon Program, link, Yellow Ribbon Program`.

On the Current VA Education Benefit Rates page, under "Current VA education benefit rates by program" > "Post-9/11 GI Bill (Chapter 33) rates," we have:

```
<a ... href="/post-911-gi-bill-chapter-33-rates" title="Post-9/11 GI Bill (Chapter 33) rates" ...>Check the current rates for the Post-9/11 GI Bill</a>
```

NVDA announces this as `Check the current rates for the Post-9/11 GI Bill, link, Post-9/11 GI Bill (Chapter 33) rates`.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Remove `title` attributes from links when they don't clarify the purpose of the link.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by shiragoodman** [comment]
@briandeconinck please update the title of this ticket - thank you!

**Comment by laflannery** [comment]
We just did some research on this and this is a core Drupal issue that is being worked on: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11843
